src/Validator/ErrorElement.php

```diff
@@ -19,6 +19,63 @@
 use Symfony\Component\Validator\ConstraintValidatorFactoryInterface;
 use Symfony\Component\Validator\Context\ExecutionContextInterface;
 
+/**
+ * @method self assertBic(array $options = [])
```

Comment:
This looks strange. Can you use something like `@mixin SomeClass`?

Reply:
Not really, because this is not a proxy.

The call method is just made the way that `assertFoo` is automatically using the `Foo` class.
So I had to add an @method with all the `Foo` constraint from symfony 4.4 prefixed with assert.